README.md

```diff
@@ -79,6 +79,7 @@ Additional lists you might find useful:
 ## Auditing / Logging
 *Plugins for auditing and logging.*
 
+- [Audit Stash plugin](https://github.com/lorenzo/audit-stash) - Flexible and rock solid audit log tracking.
```

Comment:
Going by the `-` in url I thought it should be two words :)  But it seems the readme also uses `AuditStash`.